index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./index.css">
+  <title>Awesome Books</title>
+</head>
+<body>
+  <header>
+    <h2 class="title_main">Awesome Books</h2>
+    <nav>
+      <ul>
+        <li><a href="#" id="list">List</a></li>
+        <li class="separate"></li>
+        <li><a href="#" id="add-book">Add Book</a></li>
+        <li class="separate"></li>
+        <li><a href="#" id="contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <p id="date" class="time"></p>
+  <h2 class="title_list">All awesome books</h2>
```

Comment:
- [ ] [OPTIONAL] Kindly note that this text should not be visible when clicking on all pages. Kindly consider hiding it when the user clicks on another page.